README.md

```diff
@@ -3190,6 +3221,20 @@ Other Style Guides
       inventorOf,
       ...heroArgs
     );
+
+    // bad
+    import {
+      firstVariable,
+      secondVariable,
+      thirdVariable
+    } from './someModule';
+
+    // good
+    import {
+      firstVariable,
+      secondVariable,
+      thirdVariable,
+    } from './someModule';
```

Comment:
these are identical. why is one bad and the other good?

Reply:
Trailing comma in good example